boards/DIM/Src/Io/Io_ErrorTable.c

```diff
@@ -0,0 +1,103 @@
+#include <stdint.h>
+#include <assert.h>
+#include "App_CanRx.h"
+#include "App_CanMsgs.h"
+#include "Io_SharedCanMsg.h"
+#include "Io_ErrorTable.h"
+
+// This should be used in both CAN TX and CAN RX callbacks
+void Io_ErrorTable_Update(
```

Comment:
If we're generating the code, I was considering generating `Io_ErrorTable_UpdateCanTx` and `Io_ErrorTable_UpdateRx`.

If we're generating code for BMS, the `Io_ErrorTable_UpdateCanTx` would handle only BMS errors and `Io_ErrorTable_UpdateRx` would handle errors from the rest of the boards.